### PR TITLE
feat: Remove dependabot commits from changelog

### PR DESCRIPTION
**Pull request by Jimlinz**
Changelog typically outlines what has changed since the last release. We can GitHub to automatically generate release notes every time we do a release; however, this is often filled with commits created by dependabot (bump package version). This PR excludes dependabot commits from future release notes. 

## Reference
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# .github/release.yml
+
+changelog:
+  exclude:
+    authors:
+      - dependabot
+  categories:
+    - title: release-
+      labels:
+        - '*'
```
Comment (l0b0) [inline .github/release.yml:10]: I'm a bit confused by this. Doesn't this chuck every labelled PR into a category called "release-"? Why that name, and why not include unlabelled PRs?

**Review by l0b0 — APPROVED** [review]

File: .github/release.yml (hunk 1 above)
Comment (Jimlinz) [reply .github/release.yml]: Turns out I am the one who is confused about this. I was thrown off by the `Required` label in the description (GitHub documentation). Have created a separate repo to test - they were indeed not needed. It is only required if I wanted to include categories in the changelog. The documentation could have been a bit clearer, or maybe the user could be a bit better (i.e. me 😅 )

File: .github/release.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# .github/release.yml
```
Comment (l0b0) [inline .github/release.yml:1]: I guess this is a copy-pasta error from the docs?

**Review by l0b0 — APPROVED** [review]

File: .github/release.yml (hunk 1 above)
Comment (l0b0) [reply .github/release.yml]: Very cool that you're testing in a separate repo!

**Pull request merged**